protocol/0007-POSN-non_core_positions_api.md

```diff
@@ -45,9 +45,19 @@ The API is expected to expose:
 ### Position
 
 - Open volume (this is a core API)
-- Unrealised P&L (method = averaged)
-- Realised P&L (method = averaged)
 - Open volume average entry price (method = averaged)
+- Unrealised P&L (method = averaged)
+- Realised P&L for the market for the key (method = averaged)
+- Realised P&L for the market for the key including fees(method = averaged)
+- Realised P&L for the market for the key inclding fundings (method = averaged)
+- Realised P&L for the market for the key including both fees and fundings (method = averaged)
+- Realised P&L for the market for the key since the position last changed from 0 (method = averaged)
```

Comment:
And the other two as well, so this including fees and this including funding. 

It should probably be "since the last time the position changed from `0` or flipped sign."